anaconda_linter/lint/check_syntax.py

```diff
@@ -23,6 +23,20 @@ class version_constraints_missing_whitespace(LintCheck):
 
```

Comment:
```suggestion
!!!! THIS IS A TEST COMMENT, DO NOT COMMIT !!!!
```
DO NOT COMMENT THIS CHANGE!